packages/mongo-nav/src/MongoNav.spec.tsx

```diff
@@ -40,17 +40,13 @@ describe('packages/mongo-nav', () => {
     expectedElements.charts = queryByTestId('project-nav-charts');
   };
 
-  let onOrganizationChange: jest.Mock;
```

Comment:
We should have coverage that this API usage works as expected. Where a consumer can call `setValue` and it works.

Reply:
👍